guide/12-enrich-data-with-thematic-information/part4_discover_and_enrich_standard_geographies.ipynb

```diff
@@ -0,0 +1,1208 @@
+{
```

Comment:
<p>This method, since little more than a wrapper for the standard geography query REST endpoint, there is more explanation of the options for the <code>geoquery</code> parameter under</p><p> </p><p>This sentence doesn't make sense to me</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/1309/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>